### PR TITLE
changed times week 2

### PR DESCRIPTION
**Pull request by chrbeckm**
at least not weed 2

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ Die Links zu den Zoom-Räumen werden an die angemeldeten Personen per E-Mail ges
    <!---/ Umstieg auf Linux für Interessenten--->
    </td>
   </tr>
+ <tr>
+  <td></td>
```
Comment (maxnoe) [inline index.md:118]: Why the empty cells?

Comment (chrbeckm) [reply index.md:118]: split the weeks, same below between first and second week

Comment (maxnoe) [reply index.md:118]: I don't think you need empty cells for that. An empty row should be enough

Comment (maxnoe) [reply index.md:118]: or add `style="margin-top: 10pt;"` to the row that should have more space above

Comment (chrbeckm) [reply index.md:118]: you need the empty cells, empty row only doesn't work

Comment (chrbeckm) [reply index.md:118]: i want the double lines to separate the weeks

Comment (maxnoe) [reply index.md:118]: What about ` <td colspan="4"></td>`  ?

**Review by maxnoe — APPROVED** [review]

**Pull request merged**